aten/src/ATen/native/native_functions.yaml

```diff
@@ -1316,6 +1316,16 @@
 
 - func: group_norm(Tensor input, int num_groups, Tensor? weight=None, Tensor? bias=None, float eps=1e-05, bool cudnn_enabled=True) -> Tensor
 
+- func: native_group_norm(Tensor input, Tensor? weight, Tensor? bias, int N, int C, int HxW, int group, float eps) -> (Tensor, Tensor, Tensor)
+  dispatch:
```

Comment:
do you need to define dispatch now? native_group_norm is not device-specific, it should work without dispatch.